e2e-tests/api-tests/src/test/java/org/eclipse/edc/identityhub/tests/PresentationApiEndToEndTest.java

```diff
@@ -543,21 +544,23 @@ class Postgres extends Tests {
         private static final Integer DB_PORT = getFreePort();
 
         @RegisterExtension
+        @Order(1)
         static IdentityHubCustomizableEndToEndExtension runtime;
         static PostgresSqlService server = new PostgresSqlService(DB_NAME, DB_PORT);
 
+        @Order(0) // must be the first extension to be evaluated since it starts the DB server
+        @RegisterExtension
```

Comment:
Given that there's also the `afterAll` behavior it could be better to extract a different an extension class that implements both `BeforeAllCallback` and `AfterAllCallback`, so the correct order will be applied also on the tear down phase

Reply:
i think the `afterAll` is not even needed, since the testcontainers wind down on their own

Reply:
When used without the `@Container` annotation I'm not sure, especially in this case where the container is static (sources: [junit5](https://java.testcontainers.org/test_framework_integration/junit_5/), [manual container lifecycle](https://java.testcontainers.org/test_framework_integration/manual_lifecycle_control/))

Reply:
just tried with both the `@Container` annotation on `PostgresSqlService#postgreSqlContainer` and without it - both times the container gets auto-stopped and even removed